docs/ref/django-admin.txt

```diff
@@ -1362,6 +1362,19 @@ files is:
     byte-compile invalid ``*.py`` files, template files ending with ``.py-tpl``
     will be renamed to ``.py``.
 
+.. _trusted_code_warning:
+
+.. warning::
+
+   The contents of custom app (or project) templates should always be
+   audited before use: Such templates define code that will become
```

Comment:
a colon is not normally understood to start a new sentence
```suggestion
   audited before use: such templates define code that will become
```

Reply:
This seems to be a British-vs-American English issue, see e.g.
https://www.grammarly.com/blog/capitalization-after-colons/

Reply:
😲🇺🇸